### ISSUE TITLE
Problem with special characters in object names

### ISSUE THREAD

**Issue by ilukac** [mghoneimy/php-graphql-oqm#8]
Found a small problems with underscore. 

The endpoint is providing the "_foobar" and "foobar" names of objects.  The the generated code creates 2 selectFoobar() functions, hence its an error.

Not sure how to tackle this. Maybe replace the underscore with some character combination?

**Comment by mghoneimy** [comment]
Hello @ilukac ,
While I do realize now that this is a problem, I feel like it's not that common. Fixing it will require big changes to how the classes are generated.
I doubt that there's a character combination that can be used to replace the underscores that won't make the problem worse.
I'll look into this over the weekend and update you on what can be done.

**Comment by ilukac** [comment]
Yeah, its not a big issue, I can easily fix the problem in the generated code, but submitted this so you are aware of the problem. I know its not easy to solve it :/
